src/models/users.js

```diff
@@ -0,0 +1,8 @@
+import { getUuid } from '../utils/numbers';
+import icons from '../constants/icons';
+
+export const createUser = (name, iconId) => ({
```

Comment:
icons dont have id

Reply:
I know, I can rename it, I just thought that it's not important at all (from model perspective).

Reply:
well after reading this code i can assume that icons are objects, not just plain string